src/plugins/intel_cpu/src/plugin.h

```diff
@@ -46,6 +46,19 @@ class Engine : public InferenceEngine::IInferencePlugin {
 
     void ApplyPerformanceHints(std::map<std::string, std::string> &config, const std::shared_ptr<ngraph::Function>& ngraphFunc) const;
 
+    struct StreamCfg {
+        int num_streams;
```

Comment:
Why we need separate "num_streams" field? Maybe we can use "big_core_streams" and "threads_per_stream_big" in case the system is not hybrid.

Reply:
This change will happen in following PRs after we implement searching resource in CPU mapping table instead of calculating resource based on core offset.